pkg/acceptance/BUILD.bazel

```diff
@@ -27,16 +32,18 @@ go_library(
 go_test(
     name = "acceptance_test",
     size = "enormous",
+    visibility = ["//build/bazelutil:__subpackages__"],
     srcs = [
         "adapter_test.go",
         "cli_test.go",
         "compose_test.go",
         "debug_remote_test.go",
         "main_test.go",
     ],
-    data = glob(["testdata/**"]),
+    gotags = ["acceptance"],
+    data = glob(["testdata/**"]) + ["//pkg/cmd/cockroach-short:cockroach-short"],
```

Comment:
Seems wrong, you're building `cockroach-short` twice (once manually, and then once implicitly here).

Reply:
IIRC, the main idea here was to have proper dependencies set. The manual built should be almost instantaneous?